templates/kubernetes-cluster-with-new-vpc.template

```diff
@@ -521,6 +522,10 @@ Resources:
             curl -s -o $BASTION_BOOTSTRAP_FILE $BASTION_BOOTSTRAP
             chmod +x $BASTION_BOOTSTRAP_FILE
 
+            # This gets us far enough in the bastion script to be useful.
+            apt-get -y update && apt-get -y install python-pip
+            pip install --upgrade pip &> /dev/null
```

Comment:
Isn't this runtime, not pre-baked?

Reply:
bingo. you are correct. https://github.com/aws-quickstart/quickstart-linux-bastion/blob/master/templates/linux-bastion.template#L711